frontend/config/webpack.config.common.js

```diff
@@ -0,0 +1,34 @@
+'use strict'
+
+const path = require('path');
```

Comment:
__[ESLint]__ Error - Parsing error: The keyword 'const' is reserved ([link](https://sider.review/gh/repos/108558043/pulls/347#issue-4065185))

<sub>posted by Sider</sub>